nvflare/app_common/workflows/cross_site_model_eval.py

```diff
@@ -128,7 +128,6 @@ def start_controller(self, fl_ctx: FLContext):
 
         # If the list of participating clients is not provided, include all clients currently available.
         if not self._participating_clients:
-            engine.sync_clients_from_main_process()
             clients = engine.get_clients()
```

Comment:
Does this call guarantee that only clients applicable to the job are returned?

Reply:
Yes, this only include the clients running the current job.